### ISSUE TITLE
WCAG 1.4.12 🟠 (Partially Supports): Text Spacing

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5120]
## WCAG 1.4.12: Text Spacing

**Level:** AA
**Principle:** Perceivable
**Result:** 🟠  *Partially Supports*
Details:
- OLH 🟠  Partially Supports
- Clean 🟠  Partially Supports
- Material 
  - desktop width  :x: Does not Support
  - mobile width 🟠  Partially Supports

### Resources
[WCAG 2.2 Success Criterion 1.4.12](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html)

> In content implemented using markup languages that support the following [text](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html#dfn-text) [style properties](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html#dfn-style-property), no loss of content or functionality occurs by setting all of the following and by changing no other style property:

> - Line height (line spacing) to at least 1.5 times the font size;
> - Spacing following paragraphs to at least 2 times the font size;
> - Letter spacing (tracking) to at least 0.12 times the font size;
> - Word spacing to at least 0.16 times the font size.

> Exception: [Human languages](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html#dfn-human-language) and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.

>Note 1
Content is not required to use these text spacing values. The requirement is to ensure that when a user overrides the authored text spacing, content or functionality is not lost.

>Note 2
Writing systems for some languages use different text spacing settings, such as paragraph start indent. Authors are encouraged to follow locally available guidance for improving readability and legibility of text in their writing system.

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
#4127
> ❌ text spacing
There are a handful of places where text overflows its container when the spacing is adjusted. There are a couple of bugs which are more likely at higher text spacing.

 ### Progress since last audit
- #4260 
- #4230 
- #755 
- #4261  
- #4264
- #4265 
- #4304

### Automated Testing
No violations were found for the axe rule "avoid-inline-spacing" which tests whether inline spacing is adjustable with custom stylesheets.

 ### Audit Notes

Our themes are intended to allow for text spacing, but there are a handful of places where this has not yet been achieved such that none of our three themes are compliant.  

The article page works across all three themes on both mobile and desktop widths.

For OLH and Clean, the bugs affect isolated pieces of content, and so these two themes partially support.  

However for Material the main navigation is not usable at higher text spacing which have a significant impact on any user who needed this feature, and therefore Material does not support it.   The mobile navigation does work, and is available to desktop users by reducing their window size.  This or the accessibility mode could be signposted to users as an alternative.

 ### Work arising

- #5135
